lints/lint_rsa_public_exponent_too_small.go

```diff
@@ -26,8 +26,11 @@ func (l *rsaParsedTestsExpBounds) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rsaParsedTestsExpBounds) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	theKey := c.PublicKey.(*rsa.PublicKey)
-	if theKey.E >= 3 { //If Cmp returns 1, means N > E
+	key, ok := c.PublicKey.(*rsa.PublicKey)
```

Comment:
This is stated in CAB BR 6.1.6
RSA: The CA SHALL confirm that the value of the public exponent is an odd number equal to 3 or more. 
Additionally, the public exponent SHOULD be in the range between 2^16+1 and 2^256‐1